x-pack/plugins/lens/tsconfig.json

```diff
@@ -61,6 +61,7 @@
     "@kbn/monaco",
     "@kbn/language-documentation-popover",
     "@kbn/core-saved-objects-common",
+    "@kbn/core-lifecycle-browser",
```

Comment:
We are using this for importing the CoreStart. But this is already available in Lens? Or maybe I am missing something here?

Reply:
Good catch. Will look into this!